src/server/physics/marbles.js

```diff
@@ -1,10 +1,39 @@
-module.exports = function(Ammo, world) {
+module.exports = function(Ammo, world, mapBuilder) {
+	let _marblesTransformAux = new Ammo.btTransform(),
+		_pos = undefined,
+		_rot = undefined,
+		_id = 0,
+		_finishedMarbles = [];
+
+	let _randomPositionInStartAreas = function() {
+		let startAreas = mapBuilder.getStartAreas();
+		let area = startAreas[Math.floor(startAreas.length * Math.random())];
+
+		let transform = new Ammo.btTransform();
+		transform.setIdentity();
+		transform.setOrigin(
+			new Ammo.btVector3(
+				Math.random() * area.prefabEntity.colliderData.width  - ( area.prefabEntity.colliderData.width  * .5 ),
+				Math.random() * area.prefabEntity.colliderData.height - ( area.prefabEntity.colliderData.height * .5 ),
+				Math.random() * area.prefabEntity.colliderData.depth  - ( area.prefabEntity.colliderData.depth  * .5 )
+			)
+		);
+
+		// Clone the transform because op_mul modifies the transform it is called on
+		let newTransform = new Ammo.btTransform();
+		newTransform.setIdentity();
+		newTransform.setOrigin(area.transform.getOrigin());
+		newTransform.setRotation(area.transform.getRotation());
+
+		newTransform.op_mul(transform); // Modifies "newTransform"
+
+		let origin = newTransform.getOrigin();
+
+		return origin;
+	};
+
 	return {
 		list: [],
-		_marblesTransformAux: new Ammo.btTransform(),
-		_pos: undefined,
-		_rot: undefined,
-		_id: 0,
 
 		createMarble(meta) {
 			// Create physics body
```

Comment:
Digression: We should have the size randomness defined in `meta`, not in this function.

Reply:
I'll leave that for #70 ;)